Changing text color feature

Mir3605 opened the pull request:
I added the possibility to change both title text and message text color. I also changed the color of the background of GuideMessageView to transparent, as it caused weird behavior with non-white layouts.

MCzech3605 committed:
Changing text color feature

mreram commented:
Hi @Mir3605,

Thanks a million for your contribution!

Could you please provide an example in the sample app too? and would be nice if you could provide a screenshot of weird behavior with non-white layouts as you mentioned!

Thanks in advance!

MCzech3605 committed:
Background setter created and sample app activity inserted

Mir3605 commented:
Hi @mreram !

Sure! I added a sample of these functionalities to the sample app. 
I have also created setter for the background color, as I observed that transparent background doesn't work well with the sample app. 

Strangely, I cannot repeat the bug with weird behavior in a non-white background. However, it was identical to the issue 42: https://github.com/mreram/ShowCaseView/issues/42

I have also updated the readme file with new coloring functionalities.
